lib/auth/Vault.js

```diff
@@ -24,6 +24,12 @@ function vaultSignatureCb(err, authInfo, log, callback, streamingV4Params) {
     const info = authInfo.message.body;
     const userInfo = new AuthInfo(info.userInfo);
     const authorizationResults = info.authorizationResults;
+    const auditLog = { accountDisplayName: userInfo.getAccountDisplayName() };
+    const iamDisplayName = userInfo.getIAMdisplayName();
+    if (iamDisplayName) {
```

Comment:
wondering if it would be beneficial to list the display name even if it is null, for logging purposes. Not sure how helpful that would be though.

Reply:
Good point! The requirement for the customer is only username, I added account name cuz I thought that was important. Adding user name as null will just extra add extra processing for the of leef log converter.  We already log too much in info :) 
So that’s why I didn’t add it .